api-proxy/src/reqhandler.mjs

```diff
@@ -0,0 +1,136 @@
+import * as fs from 'fs'
+const fsp = fs.promises
+import * as https from 'https'
+import * as stream from 'stream'
+import * as nu from 'util'
+import * as zlib from 'zlib'
+
+const omit = ks => obj =>
+  Object.keys(obj).reduce((acc, k) => ks.includes(k) ? acc : ({...acc, [k]: obj[k]}), {})
+
+const promiseEvent = eventName => x => new Promise(resolve => {
+  x.on(eventName, a => resolve(a))
+})
+
+const quoteSingles = x => x.replace(/[']/g, '\\\'')
+const quoteBackquotes = x => x.replace(/[`]/g, '\\`')
+const formatObj = x => nu.inspect(x, {depth: Infinity, maxArrayLength: 10e3, breakLength: 100})
+
+const jsonParse = s => {
```

Comment:
This is like a Left(error) / Right(success) but formatResBody has the opposite semantics.  Is there a reason for that?

Reply:
I like that. It didn't occur to me that the string in `formatResBody` was more of a "left," but I agree and changing it would make things more consistent.